### PR TITLE
html-forms - Michael Suh 

### PR DESCRIPTION
**Pull request by michael-suh**
html forms screenrecord

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+
+  <h1>Coding Form!</h1>
+  <label for="email">Email</label>
+  <div><input type="email" name="name" id="name"></div>
```
Comment (codingcodymiller) [inline html-forms/index.html:13]: The value of the `id` attribute on the `input` needs to match the value of the `for` attribute on the `label` it corresponds with.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-forms/index.html [hunk 2]
```diff
@@ -9,30 +9,46 @@
...
+    <div>
+      <input type="email" name="name" id="email">
+    </div>
 
 
-    <p>Favorite Language (choose one)</p>
-    <input type="radio" name="language" value="HTML">HTML
-    <input type="radio" name="language" value="CSS">CSS
-    <input type="radio" name="language" value="JS">JavaScript
+    <label for="reasons"></label>Why is coding cool?
```
Comment (codingcodymiller) [inline html-forms/index.html:19]: Why isn't this text in the `label`?

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+
+    <p>Favorite Language (choose one)</p>
+    <input type="radio" name="language" value="HTML">HTML
+    <input type="radio" name="language" value="CSS">CSS
+    <input type="radio" name="language" value="JS">JavaScript
+
+
+    <p>Would you like to sign up for more information?</p>
+    <input type="checkbox" name="signup" value="signmeup" /> Sign me up!
```
[the remark this answers is not in the document]
Comment (codingcodymiller) [reply html-forms/index.html:46]: This is still not resolved, the "Sign me up!" text needs to be _in_ the `label` element

File: html-forms/index.html [hunk 4]
```diff
@@ -9,30 +9,46 @@
...
+
+    <label for="signmeup"></label>
     <p>Would you like to sign up for more information?</p>
     <input type="checkbox" name="signup" value="signmeup" /> Sign me up!
 
 
-    <div>Comments</div>
+
     <label for="comments"></label>
+    <div>Comments</div>
```
Comment (codingcodymiller) [inline html-forms/index.html:51]: This text should be in the `label` above this, not in a `div`

From the exercise instructions:
image

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by michael-suh** [comment]
html forms screenrecord

File: html-forms/index.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (michael-suh) [reply html-forms/index.html:46]: resolved now

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**